test/lit.cfg

```diff
@@ -425,7 +425,7 @@ if platform.system() == 'Darwin' and (run_os == 'macosx' or run_os == 'darwin'):
     # <rdar://problem/16678410> Adding more libraries with -lfoo to REPL is broken
     config.available_features.add('swift_repl')
     config.available_features.add('swift_interpreter')
-elif platform.system() == 'Linux':
+elif platform.system() == 'Linux' and not (platform.processor().startswith('armv6') or platform.processor().startswith('armv7')):
```

Comment:
@tienex For reference, a better solution than using `-Bsymbolic` long term might be to have swiftc emit public ELF symbols with `protected` visibility (or at least metadata symbols we relatively-reference), which should have the same effect. I filed https://bugs.swift.org/browse/SR-667